packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.controller.js

```diff
@@ -641,7 +641,7 @@ export default /* @ngInject */
     }, 300);
   }
   if ($location.search().action === 'mitigation' && $location.search().ip) {
-    $scope.trackPage(`${TRACKING_PREFIX}::enable-permanent-mitigation`);
+    trackPage(`${TRACKING_PREFIX}::enable-permanent-mitigation`);
```

Comment:
it's not `atInternet.trackPage` ? Or is there a wrapper method available in the controller ?

Reply:
Injection made in the controller is `trackPage` due to https://github.com/ovh/manager/blob/93246e8c13a19278589ee511aa5954f17ca4f4eb/packages/manager/apps/dedicated/client/app/ip/ip.routing.js#L15